src/lib/tutorials/normalize-product-slug.ts

```diff
@@ -0,0 +1,45 @@
+import { LearnProductSlug, ProductSlug } from 'types/products'
+import { isProductSlug } from 'lib/products'
+import { isProductOption, ThemeOption } from 'lib/learn-client/types'
+
+/**
+ * Given a "product" slug from a Tutorials context,
+ * such as in part of a collection or tutorial slug,
+ * Return a "product" slug compatible with Dev Dot's ProductSlug type.
+ *
+ * This is very specifically targeted at normalizing "cloud" to "hcp".
+ * In Tutorials contexts, we use "cloud"; in Dev Dot we use "hcp".
+ */
+export function normalizeProductSlugForDevDot(slug: string): ProductSlug {
```

Comment:
In Dev Dot, we treat `hcp` as a product. In Tutorials land, we kind of do in a way as well, but we use the slug `cloud`. These functions are meant as a single source of truth for normalizing between the two.